xml2rfc/writers/pdf.py

```diff
@@ -123,14 +123,14 @@ def write(self, filename):
     def get_serif_fonts(self):
         fonts = set()
         scripts = self.root.get('scripts').split(',')
```

Comment:
Since you're here, may as well convert the whole method to scratch the itch you felt in the first place.
But don't hold up a release over it.